keps/sig-node/1797-configure-fqdn-as-hostname-for-pods/README.md

```diff
@@ -626,17 +628,19 @@ _This section must be completed when targeting beta graduation to a release._
 
 _This section must be completed when targeting beta graduation to a release._
 
-TODO
 
 * **How can an operator determine if the feature is in use by workloads?**
   Ideally, this should be a metrics. Operations against Kubernetes API (e.g.
   checking if there are objects with field X set) may be last resort. Avoid
   logs or events for this purpose.
 
+  For a pod to be using this feature it needs to define the PodSpec fields
```

Comment:
nit: let's use the wording that is used here:
https://github.com/kubernetes/enhancements/commit/4547434481f5639e5a14ab01e510418d6ac9cd73#diff-8feaa96d605066ba063646f4d18e02f5R241